src/ui/inspector/PlotElementsPool.vue

```diff
@@ -0,0 +1,333 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+<template>
+<div class="c-elements-pool">
+    <Search
+        class="c-elements-pool__search"
+        :value="currentSearch"
+        @input="applySearch"
+        @clear="applySearch"
+    />
+    <div
+        class="c-elements-pool__elements"
+    >
+        <ul
+            v-if="yAxis1.length > 0
+                || yAxis2.length > 0
+                || yAxis3.length > 0"
+            id="inspector-elements-tree"
+            class="c-tree c-elements-pool__tree"
+        >
+            <element-item-group
+                :parent-object="parentObject"
+                :allow-drop="allowDrop"
+                label="Y Axis 1"
+                @drop-group="moveTo($event, 0, Y_AXIS_1)"
+            >
+                <li
+                    class="js-first-place"
+                    @drop="moveTo($event, 0, Y_AXIS_1)"
+                ></li>
+                <element-item
+                    v-for="(element, index) in yAxis1"
+                    :key="element.identifier.key"
+                    :index="index"
+                    :element-object="element"
+                    :parent-object="parentObject"
+                    :allow-drop="allowDrop"
+                    @dragstart-custom="moveFrom($event, Y_AXIS_1)"
+                    @drop-custom="moveTo($event, index, Y_AXIS_1)"
+                />
+                <li
+                    v-if="yAxis1.length > 0"
+                    class="js-last-place"
+                    @drop="moveTo($event, yAxis1.length, Y_AXIS_1)"
+                ></li>
+            </element-item-group>
+            <element-item-group
```

Comment:
Remove duplicate code here for `element-item-group` and `element-item` components.
See above